template/main.tpl

```diff
@@ -5,6 +5,7 @@
     <title>Clonkspot {$l->s('league')}</title>
     <link rel="stylesheet" type="text/css" href="/css/clonkspot.css">
     <link rel="stylesheet" type="text/css" href="{$base_path}league.css">
+    <script src="js/date-offset.js"></script>
```

Comment:
Probably needs `{$base_path}` here like the stylesheet above.

Reply:
Should be done, I didn't use base_path before, tested with my config.php trailing and leading slash.